apps/web/src/pages/quick-start/components/demo-flow/DigestDemoFlow.tsx

```diff
@@ -0,0 +1,92 @@
+import React from 'react';
+import styled from '@emotion/styled';
+import { useMantineColorScheme } from '@mantine/core';
+import ReactFlow, { Edge, Node } from 'react-flow-renderer';
+
+import { colors } from '../../../../design-system';
+import { TriggerNode } from './nodes/TriggerNode';
+import { DigestNode } from './nodes/DigestNode';
+import { EmailNode } from './nodes/EmailNode';
+
+export function DigestDemoFlow() {
+  const { colorScheme } = useMantineColorScheme();
+
+  return (
+    <Wrapper dark={colorScheme === 'dark'}>
+      <div style={{ minHeight: '500px', height: '100%', width: 'inherit' }}>
```

Comment:
Please ignore the sizing here they will could change in the future, at the moment I only created this part on the main body section.